web/src/cypress/integration/profile.ts

```diff
@@ -73,6 +73,71 @@ function testProfile(): void {
       })
   })
 
+  describe('Settings', () => {
+    it('should visit profile', () => {
+      cy.visit('/')
+      cy.get('[aria-label="Manage Profile"]').click()
+      cy.get('[data-cy="manage-profile"]')
+        .find('button')
+        .contains('Manage Profile')
+        .click()
+      cy.url().should('eq', Cypress.config().baseUrl + '/profile')
+    })
+
+    it('should change the theme mode', () => {
+      cy.get('[aria-label="Manage Profile"]').click()
+      cy.get('[data-cy="manage-profile"]')
+        .find('button')
+        .contains('Light')
+        .click()
+      cy.get('div[id="app-root"]').should(
+        'have.css',
+        'background-color',
+        'rgb(211, 211, 211)',
+      )
+
+      cy.get('[data-cy="manage-profile"]')
+        .find('button')
+        .contains('Dark')
+        .click()
+      cy.get('div[id="app-root"]').should(
+        'have.css',
+        'background-color',
+        'rgb(18, 18, 18)',
+      )
```

Comment:
These seem fragile and not helpful if say we change the mode as the test would break even if it's behaving correctly.

Could we just test that the color changes? (E.g. read current, click button, assert a change, click previous button, assert same as before)

Reply:
I guess so, we can do that

Reply:
Ill have to see if Cypress can do that effectively.

Reply:
Resolved in most recent commit